packages/e2e-tests/helpers/pause-information-panel.ts

```diff
@@ -352,7 +352,12 @@ export async function waitForPaused(page: Page, line?: number): Promise<void> {
   }
 }
 
-export async function waitForScopeValue(page: Page, name: string, expectedValue: Expected) {
+export async function waitForScopeValue(
+  page: Page,
+  name: string,
+  expectedValue: Expected,
+  expandScopes = true
```

Comment:
What would it mean to call this function _without_ this value being true? I think it would mean we'd be stuck waiting forever for a value that wasn't in the HTML. Seems bad?

Reply:
It means that the function doesn't try to expand anything, in this case it will only succeed if the scopes are already expanded - which is exactly what I wanted to test (see the changes in object_preview-03).

Reply:
I guess that makes sense, but I think maybe we should use a different method for that– an _expect_ method, rather than a _wait for_ method. I guess this is kind of subjective but I think it would fit a little more with our other existing methods.